qemu/tests/cfg/virtio_fs_share_data_hugepage.cfg

```diff
@@ -0,0 +1,84 @@
+    type = virtio_fs_share_data
+    virt_test_type = qemu
+    required_qemu = [4.2.0,)
+    kill_vm = yes
+    start_vm = no
+    filesystems = fs
+    fs_driver = virtio-fs
+    fs_source_type = mount
+    fs_source_dir = virtio_fs_test/
+    force_create_fs_source = yes
+    remove_fs_source = yes
+    fs_target = 'myfs'
+    fs_driver_props = {"queue-size": 1024}
+    # memory backend setting
+    setup_hugepages = yes
+    pre_command_noncritical = yes
+    pre_command = 'sync && echo 3 > /proc/sys/vm/drop_caches && echo 1 > /proc/sys/vm/compact_memory'
+    vm_mem_share = yes
+    io_timeout = 600
+    test_file = 'test_file'
+    fs_dest = '/mnt/${fs_target}'
+    driver_name = viofs
+    fs_binary_extra_options = ''
+    Windows:
+        # install winfsp tool
+        i386, i686:
+            install_path = 'C:\Program Files'
+        x86_64:
+            install_path = 'C:\Program Files (x86)'
+        install_cmd = 'msiexec /i WIN_UTILS:\winfsp-1.7.20172.msi /qn'
+        check_installed_cmd = 'dir "%s" |findstr /I winfsp'
+        viofs_log_file = C:\\viofs_log.txt
+        viofs_sc_create_cmd = 'sc create VirtioFsSvc binpath="%s -d -1 -D ${viofs_log_file}" start=auto'
+        viofs_sc_create_cmd += ' depend="WinFsp.Launcher/VirtioFsDrv" DisplayName="Virtio FS Service"'
+        viofs_sc_start_cmd = 'sc start VirtioFsSvc'
+        viofs_sc_query_cmd = 'sc query VirtioFsSvc'
+        virtio_win_media_type = iso
+        cdroms += " virtio"
+    variants:
+        - with_cache:
+            variants:
+                - auto:
+                    fs_binary_extra_options += ",cache=auto"
+                - always:
+                    fs_binary_extra_options += ",cache=always"
+                - none:
+                    fs_binary_extra_options += ",cache=none"
+    variants:
+        - basic_test:
+            cmd_dd = 'dd if=/dev/urandom of=%s bs=1M count=2048 oflag=direct'
+            cmd_md5 = 'md5sum %s'
+            folder_test = 'yes'
+            test_folder = 'test_folder'
+            cmd_new_folder = "cd %s && mkdir -p ${test_folder}"
+            cmd_copy_file = "cp ${test_file} ${test_folder}"
+            cmd_rename_folder = "mv ${test_folder} ${test_folder}_tmp"
+            cmd_del_folder = "rm -rf ${test_folder}_tmp"
+            cmd_check_folder = "ls -al ${test_folder}_tmp"
+            io_timeout = 120
+            s390, s390x:
+                cmd_dd = 'dd if=/dev/urandom of=%s bs=1M count=200 oflag=direct'
+            Windows:
+                cmd_dd = 'dd if=/dev/random of=%s bs=1M count=200'
+                cmd_md5 = "%s: && md5sum.exe %s"
+                cmd_new_folder = "%s && md ${test_folder}"
+                cmd_copy_file = "copy ${test_file} ${test_folder}"
+                cmd_rename_folder = "rename ${test_folder} ${test_folder}_tmp"
+                cmd_del_folder = "rd /q /s ${test_folder}_tmp"
+                cmd_check_folder = "dir ${test_folder}_tmp"
+    variants:
+        - @default:
```

Comment:
The default mode is already included in other cases, do we still need this option?

Reply:
There is only one default in thie cfg file,I think it has no relation with other cfg files.